woocommerce/woocommerce-ios
# Migration back from woo.com to woocommerce.com in URLs

Closes: #12430

## Description
It has been decided to migrate back from woo.com to woocommerce.com. For this reason, we should update the URLs and some contents in the app to point to the new domain, rather than go through the redirect.

More info on this decision here: peaMlT-w5


## Testing instructions
These changes are widespread, and it doesn't seem practical to verify every link within the app. As I've reviewed them, I've opened each one in a web browser to ensure they all lead to the correct destinations. Currently, a redirect is active. I would suggest spot-checking a selection of the changes and also testing some within the app.

---
